### DISCUSSION TITLE
Beets treating different recordings as duplicates

### DISCUSSION THREAD

**Discussion by BGarber42** [beetbox/beets#5206] [Q&A] [answered]
I'm trying to import tracks from [an album](https://musicbrainz.org/release/5653c411-c00d-4735-b243-d489161469f9)

And when I get to trying to import track [4.1](https://musicbrainz.org/release/5653c411-c00d-4735-b243-d489161469f9/disc/4#73086469-f191-4fac-aca1-8d987854a218) it thinks it's the same as track [1.1](https://musicbrainz.org/release/5653c411-c00d-4735-b243-d489161469f9/disc/1#e71982e2-c8d8-4e9a-a728-839695a82131) even trying to import it as singletons and specifying the recording ID.

How do I fix this?

**Comment by add-n2x** [comment]
How have you configured matching duplicates?

By default the MusicBrainz Track ID (`mb_trackid`)and Album ID (`mb_albumid`) is used. So it should not match as duplicate in my understanding.

**Reply by DjSlash** [reply]
This is not a `duplicates` plugin issue. :)

**Comment by DjSlash** [comment, marked as answer]
~~Try enabling the `--from_scratch` option.~~ Sorry, that is my head confusing things.

 It may be that existing metadata is confusing the matching. Try to see if that might be the case (with the `info` plugin loaded, you can see with `beet info /path/to/file.mp3`
